html/changelogs/Paprka-crades.yml.txt

```diff
@@ -0,0 +1,6 @@
+author: Paprka
```

Comment:
Just set this to whatever you want if you want to change the change log author, i.e author: AstralEnigma

You cannot have two, but you could add something in changes -tweak: "Based on work by paprka"